packages/strapi-admin/admin/src/translations/fr.json

```diff
@@ -2,7 +2,7 @@
   "Analytics": "Statistiques",
   "Auth & Permissions": "Auth & Permissions",
   "Content Manager": "Content Manager",
-  "Content Type Builder": "Content Type Builder",
+  "Content Type Builder": "Types Builder",
```

Comment:
Keep the old one

Reply:
Without a dash between "Content" and "Type"?